src/pages/Vault/index.tsx

```diff
@@ -110,16 +130,16 @@ const Vault = (): JSX.Element => {
           apyScore,
           issuableAmount
         ] = await Promise.allSettled([
-          window.bridge.vaults.get(vaultId, collateralIdLiteral),
+          window.bridge.vaults.get(vaultAccountId, COLLATERAL_ID_LITERAL),
           window.bridge.vaults.getWrappedReward(
-            newAccountId(window.bridge.api, selectedVaultAddress),
```

Comment:
@theodorebugnet @nud3l 
I used `window.bridge.api.createType(ACCOUNT_ID_TYPE_NAME, selectedVaultAccountAddress)` instead of `window.bridge.api.createType(ACCOUNT_ID_TYPE_NAME, selectedVaultAccountAddress)` to make it simpler. Please correct me if I'm wrong and they are different.

Reply:
They are the same, but I would recommend using `newAccountId` everywhere to avoid maintaining constants like `ACCOUNT_ID_TYPE_NAME` in the UI

Reply:
@theodorebugnet 
Thank you for clarifying.
@savudani8 
Thank you for your suggestion. It makes more sense to use `newAccountId`. I'll update it.